go/interpreter/lfvm/gas_test.go

```diff
@@ -59,3 +59,17 @@ func TestGas_CallGasCalculation(t *testing.T) {
 		})
 	}
 }
+
+func TestGas_CheckGasCostsForAllExecutableOpcodes(t *testing.T) {
+	for i := 0; i < int(NUM_OPCODES); i++ {
+
+		if static_gas_prices[i] == UNKNOWN_GAS_PRICE &&
+			op_2_op[i] != INVALID {
```

Comment:
nit: early bail out, `if op_2_op[i] == invalid => continue`

Reply:
nit: op_2_op is cryptic and not a good name

Reply:
`vmOp_to_lfvmOp` seems like a better, natural name, but a little more visually noisy, but I can change it to that if you approve.

Reply:
I think it is still not very good. It out of the scope of this PR. we can think about it later on.